File: code/9a069.pas
program Bai069;
var a,b,x, min: integer;
begin
  readln(a, b);
  min := a;
  if b < a then min:= b;
   for x:= 1 to min do if((a mod x = 0) and (b mod x = 0)) then write(x);
end.

File: code/9a069.cpp
#include <iostream>
using namespace std;

int main() 
{
    int a,b;
    cin>>a>>b;
    for (int x=1; x<=min(a,b); x++)
        if (a%x==0 && b%x==0)
            cout<<x<<" ";
    return 0;
}